job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy

```diff
@@ -390,6 +397,22 @@ class PublisherContext implements Context {
                                     if (data instanceof PlotPropertiesSeriesContext) {
                                         label(data.label ?: '')
                                     }
+                                    if (data instanceof PlotCSVSeriesContext) {
+                                        inclusionFlag(data.inclusionFlag)
+                                        if ((data.inclusionFlag == 'INCLUDE_BY_STRING') ||
+                                            (data.inclusionFlag == 'EXCLUDE_BY_STRING')) {
+                                            strExclusionSet {
+                                                string(data.exclusionValues ?: '')
+                                            }
+                                        }
+                                        if ((data.inclusionFlag == 'INCLUDE_BY_COLUMN') ||
+                                            (data.inclusionFlag == 'EXCLUDE_BY_COLUMN')) {
+                                            colExclusionSet()
```

Comment:
There is something missing. The indexes need to be integers and then the `colExclusionSet` is filled. That's why I suggested to use int arguments in this case...

```xml
<hudson.plugins.plot.CSVSeries>
  <file>test</file>
  <label/>
  <fileType>csv</fileType>
  <colExclusionSet>
    <int>1</int>
    <int>2</int>
    <int>6</int>
  </colExclusionSet>
  <inclusionFlag>EXCLUDE_BY_COLUMN</inclusionFlag>
  <exclusionValues>1,2,6</exclusionValues>
  <url>bar</url>
  <displayTableFlag>true</displayTableFlag>
</hudson.plugins.plot.CSVSeries>
```